### ISSUE TITLE
standalone crash when trying to load not-exit\deleted file from recent

### ISSUE THREAD

**Issue by Penime** [HolonProduction/godot_kanban_tasks#22]
The standalone version crash with error `Attempt to call function 'close' in base 'null instance' on a null instance.`  on line 75 in data/board.gd script, when I tried to open renamed\deleted\moved file from the recent (next to create board and open board on app start up) maybe make so when it's happening popup will show up with message "file deleted\renamed\moved do you want to remove it from the recent list." with yes and no buttons, and then you can remove it from the recent list if yes. and you can do the same if the file is not valid.
Maybe I will also suggest using Compatibility instead of Forward+ (I rebuild standalone in Compatibility so the app will run on old laptop like I have)
```GDScript
## Loads the data from `path` into the current instance.
func load(path: String) -> void:
	var file = FileAccess.open(path, FileAccess.READ)
	if not file:
		push_error("Error " + str(FileAccess.get_open_error()) + " while opening file for loading board data at " + path)
>>>		file.close()
		return
```

https://github.com/HolonProduction/godot_kanban_tasks/assets/122761839/157c2a8b-d43b-460b-aa2b-5723ac8a0c07

Thank you for making this amazing godot plugin\app

**Comment by HolonProduction** [comment]
Fixed with 4bdcb52321f335ad4c6daaa415a9d34fc2986690

**Comment by Penime** [comment]
Thanks a bunch :)
